### PR TITLE
[Translate ru] Index page

### PR DESCRIPTION
**Pull request by DmitrySkripunov**
[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/ru/review/495)

### CODE REVIEW

File: src/content/ru/_index.yaml [hunk 1]
```diff
@@ -0,0 +1,211 @@
+---
+book_path: "/web/_book.yaml"
+landing_page:
+  custom_css_path: "/web/styles/wf-landing.css"
+  header:
+    custom_html: "<div class=\"\\\"devsite-collapsible-section\\\"\">\n  <div class=\"\\\"devsite-header-background\\\"\">\n
```
Comment (petele) [inline src/content/ru/_index.yaml]: Escaping is completely broken.

**Comment by petele** [comment]
@sotayamashita & @chikathreesix - would you please take a look, escaping is completely broken!

**Comment by sotayamashita** [comment]
@DmitrySkripunov I have created a pull request to fix @petele is talking. 

@petele I try to monitor localization pull requests to avoid mentioning us. Sorry. :bow:

**Comment by petele** [comment]
@sotayamashita - no worries. I have a 'canned response' set up to easily tag you in as necessary. Thanks for keeping an eye on things.

**Comment by DmitrySkripunov** [comment]
Merged. @petele please, look at it.

**Comment by ilyaspiridonov** [comment]
@agektmr Reviewed, BUT I would suggest closing this PR without merging, as it's outdated. 
1) The original file has changed 
2) We have since updated the YAML processing logic on GitLocalize, so there should be no issues with escaping. 
3) I expect that meaningful translations that were contributed before can be restored via translation memory.

**Comment by agektmr** [comment]
Sorry @DmitrySkripunov we got to this pull request this late but we have to decline it.

**Pull request closed without merging**